### PR TITLE
1.x fromCallable() @Experimental -> @Beta

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Part of #3816

### CODE REVIEW

File: src/main/java/rx/Observable.java [hunk 1]
```diff
@@ -1463,7 +1463,7 @@ public void call(Subscriber<? super Object> subscriber) {
      * @see #defer(Func0)
      * @since (if this graduates from Experimental/Beta to supported, replace this parenthetical with the release number)
```
Comment (artem-zinnatullin) [inline src/main/java/rx/Observable.java]: Not sure what to set here.

**Comment by akarnokd** [comment]
`testQueueFullEmitsErrorWithVaryingBufferSize` is an unreliable test I already fixed in #3795

:+1:

**Comment by hzsweers** [comment]
:+1:

**Comment by stevegury** [comment]
👍

**Pull request merged**